rules/falco_rules.yaml

```diff
@@ -2445,7 +2445,7 @@
     Netcat runs inside container that allows remote code execution (user=%user.name user_loginuid=%user.loginuid
     command=%proc.cmdline container_id=%container.id container_name=%container.name image=%container.image.repository:%container.image.tag)
   priority: WARNING
-  tags: [network, process, mitre_execution]
+  tags: [network, process, mitre_TA0002_execution, mitre_TA0007_discovery, mitre_T1046_network_service_scanning]
```

Comment:
not `mitre_T1046_network_service_scanning`

Reply:
Solved.